### DISCUSSION TITLE
Where to see the status of a trademark related GH inquiry

### DISCUSSION THREAD

**Discussion by qknight** [community/community#56421] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Topic Area

Question

### Body

Hello,

I've filed a request in https://support.github.com/contact/trademark-policy (which was linked by https://docs.github.com/de/site-policy/content-removal-policies/github-trademark-policy) but now I don't know if that is still in progress or not.

I filed this with my qknight GH username for a organization i'm a member but not owner.

Where can i see the status of my trademark related question?

Regards,
Joachim Schiele

**Comment by ettaboyle** [comment]
Hi @qknight ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. This post is a duplicate of another that I have deleted because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. **Please do not cross-post.**

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

Thank you!
